spec/models/cat_spec.rb

```diff
@@ -1,5 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+  it "should validate name" do
+    cat = Cat.create
+    expect(cat.errors[:name]).to_not be_empty
+  end
+
+
+  
+    it "should validate age" do
+      cat = Cat.create
+      expect(cat.errors[:age]).to_not be_empty
+    end
+  
+
+   
+      it "should validate enjoys" do
+        cat = Cat.create  
+        expect(cat.errors[:enjoys]).to_not be_empty
+      end
+
+      it "should validate enjoys is at least 10 characters" do
+        cat = Cat.create 
+        expect(cat.errors[:enjoys]).to_not be_empty
+      end
+    
+
+
+        it "should validate image" do
+          cat = Cat.create
+          expect(cat.errors[:image]).to_not be_empty
+        end
+     end
```

Comment:
Looks like copy paste also copied some extra spaces. Let's clean this up and format this document before merging.